packages/prisma/migrations/20230125175109_remove_type_from_payment_and_add_app_relationship/migration.sql

```diff
@@ -0,0 +1,15 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `type` on the `Payment` table. All the data in the column will be lost.
+
+*/
+-- AlterTable
+ALTER TABLE "Payment" DROP COLUMN "type",
```

Comment:
Shouldn't be a problem since any existing payment its of type stripe.
@zomars we should do an update for every payment to update it's new appId value.

Reply:
Probably best to separate the migration into its own PR, I know @zomars recommends that, works as a prep PR for the rest of the changes, to do it in stages.

Reply:
Let's do this in this same migration

> Shouldn't be a problem since any existing payment its of type stripe.
> @zomars we should do an update for every payment to update it's new appId value.